package/crds/cloudfront.aws.crossplane.io_distributions.yaml

```diff
@@ -1606,7 +1606,7 @@ spec:
                                         configured as a website endpoint, use the
                                         CustomOriginConfig element instead.
                                       properties:
-                                        originAccessIDentity:
+                                        originAccessIdentity:
```

Comment:
Hrm, this seems like a breaking API change. Acceptable for a v1alpha1 resource (and seemingly a bug fix) but we should make note of this in the release notes.

Reply:
tested the resource via example and changed the field in the example - right we need to mention this issue as breaking API change